### ISSUE TITLE
Does not work with FF 57

### ISSUE THREAD

**Issue by bachware** [dafizilla/firefox-viewsourcewith#5]
VSW does not work with today's new FF57 (Quantum), because Mozilla now requires add-ons to be signed.  I _desperately_ need this for development speed/ease.  _**Please**_ update it ASAP, David.  I've paid for it. Thanks & ciao!

**Comment by dafi** [comment]
I suppose VSW doesn't work anymore due to the new multiprocess architecture, fixing it is almost impossible

**Comment by dpat57** [comment]
That is bad news, ViewSourceWith was an extremely useful and much-appreciated utility.  Firefox finally fixes its not responding problem but kills my productivity by deactivating my most-used add-ons, ugh.

Added:  found a workaround that allows me to edit web page content via right-click + select View Page Source, I hope this isn't rude to mention it here:
https://support.mozilla.org/en-US/questions/757430

**Comment by dafi** [comment]
Honestly I'm very angry but radical modifications require to rewrite 90% of code and many things are no longer possible with new add-ons programming model

**Comment by bachware** [comment]
Yes. I agree with the anger.  Mozilla felt they had to do what they did 
in order to keep control over add-ons' vulnerabilities, also to improve 
performance in order to remain competitive with other browsers, and in 
the process, users are thrown under the bus. Still, it's typical of 
development headaches with legacy add-ons. At least I can still have the 
VSW functionality by hacking via about:config, which is analogous to 
hacking Windows itself by using Regedit, also often necessary.  
Unfortunately, the not-so-technically adept are left helpless. A similar 
problem is Mozilla's impending abandonment of Thunderbird (it's already 
a stepchild), which is, IMHO, the most versatile and best E-mail client 
out there. Still, the new Quantum browser seems much faster, and has a 
much smaller memory footprint.

**Comment by MR57** [comment]
Shouldn't about:config and setting extensions.legacy.enabled to true allow to re-activate these add-ons? In my Firefox, they move up to the regular addon listing, but are still disabled...

**Comment by FrankConijn** [comment]
What I like most about VSW is that I get a list of CSS and JS files used on a webpage with the click of button. Which I can then review with one click as well. Does anyone know a method so that I can do that in Quantum without VSW?
